### PR TITLE
zh: add more hints for deployment on aws

### PR DESCRIPTION
**Pull request by lichunzhu**
### What is changed, added or deleted? (Required)
Add more hints for deployment on aws.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -269,6 +269,14 @@ region = us-west-21
...
     ```
 
+4. 查看 TiDB 集群启动状态：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl --kubeconfig credentials/kubeconfig_${eks_name} get pods -n ${namespace}
+    ```
+
```
Comment (ran-huang) [inline zh/deploy-on-aws-eks.md:279]: Suggest adding an expected output here, so users can confirm whether the deployment is successful or not.

Comment (lichunzhu) [reply zh/deploy-on-aws-eks.md:295]: addressed in https://github.com/pingcap/docs-tidb-operator/pull/523/commits/2285f1a658d5e2f748f3ef53eb76602752019955

File: zh/backup-to-aws-s3-using-br.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` Custom Resource (CR)
         # checksum: true
         # sendCredToTikv: true
       from:
-        host: ${tidb_host}
+        host: ${tidb_host} # ${tidb_host} 为需要导出的 TiDB 的 service name，例如 basic-tidb
```
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:189]: No need to add a description for each of them, just add the description in L321.

File: zh/backup-to-s3.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR)
       namespace: test1
     spec:
       from:
-        host: ${tidb_host}
+        host: ${tidb_host} # ${tidb_host} 为需要导出的 TiDB 的 service name，例如 basic-tidb
```
Comment (DanielZhangQD) [inline zh/backup-to-s3.md:61]: ditto

File: zh/deploy-on-aws-eks.md [hunk 4]
```diff
@@ -269,6 +269,30 @@ region = us-west-21
...
 
+4. 查看 TiDB 集群启动状态：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl --kubeconfig credentials/kubeconfig_${eks_name} get pods -n ${namespace}
+    ```
+   
+    当所有 pods 都处于 Running 状态时，则可以认为 TiDB 集群已经成功启动。一个正常运行的 TiDB 集群的案例：
```
Comment (DanielZhangQD) [inline zh/deploy-on-aws-eks.md:280]:
```suggestion
    当所有 pods 都处于 Running & Ready 状态时，则可以认为 TiDB 集群已经成功启动。一个正常运行的 TiDB 集群的案例：
```

File: zh/backup-to-aws-s3-using-br.md [hunk 5]
```diff
@@ -318,7 +318,7 @@ Amazon S3 支持以下几种 `storageClass` 类型：
 更多 `Backup` CR 字段的详细解释:
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
-* `.spec.from.host`：待备份 TiDB 集群的访问地址。
+* `.spec.from.host`：待备份 TiDB 集群的访问地址，为需要导出的 TiDB 的 service name，例如 `basic-tidb`。
```
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:321]: Please also help update `zh/backup-to-gcs.md` and `zh/backup-to-gcs-using-br.md`.

Comment (lichunzhu) [reply zh/backup-to-aws-s3-using-br.md:321]: addressed in https://github.com/pingcap/docs-tidb-operator/pull/523/commits/1f2a84c411eafd64139ce6b729e8bc7f10e974dc

**Pull request merged**